features/staging_commands.feature

```diff
@@ -90,3 +90,60 @@ Feature: Enable command behaviour when attached to an UA staging subscription
            | bionic  | bundler  |
            | focal   | ant      |
            | xenial  | jq       |
+
+    @series.lts
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Run timer script on an attached machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token_staging` with sudo
+        And I create the file `/tmp/machine-token-overlay.json` with the following:
+        """
+        {
+            "machineTokenInfo": {
+                "contractInfo": {
+                   "id": "testCID"
+                },
+                "machineId": "testMID"
+            }
+        }
+        """
+        And I create the file `/tmp/response-overlay.json` with the following:
```

Comment:
Is the backend to a point where we can have an integration test that actually talks to the backend for this feature/

Reply:
It is, the problem is that we would need to have a different test structure to properly test the real backend. We would need to compare if we had a change on the `activityToken` and `activityID`.

Also, we would need to double check with @jaycee to double check if we can use the backend for this purpose, just to double check if they have some metrics that would be affected by it.

Let's talk about this on standup

Reply:
Aha very good point! I imagine it'd be fine for metrics if we use staging for this test.

Also we could simulate a "clone" by doing the metering request twice in a row with the same activityId and activityToken I think